content/en/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md

```diff
@@ -90,10 +92,10 @@ kubectl exec -it kubernetes-downwardapi-volume-example -- sh
 In your shell, view the `labels` file:
 
 ```shell
-/# cat /etc/podinfo/labels
```

Comment:
I think our style guide mandates that the prompt be removed from command-line examples. Please review all code samples this PR modifies and make sure they don't include the prompt.

Reply:
Hi Misty, I took the prompts out in the new commit. However, it feels a little weird to have just nothing there? Should I replace it with something else? (I feel as if I'm missing something.)

Reply:
Well, it is missing the whole command now. 😂 Try something like:

````
```shell
cat /etc/podinfo/labels
```
````